pkg/webui/containers/fetch-table/index.js

```diff
@@ -353,6 +356,7 @@ class FetchTable extends Component {
                 placeholder={searchPlaceholderMessage}
                 className={style.searchBar}
                 inputWidth="full"
+                maxlength={searchQueryMaxLength}
```

Comment:
```suggestion
                maxLength={searchQueryMaxLength}
```

Reply:
[`maxlength` is an HTML attribute](https://www.w3schools.com/tags/att_input_maxlength.asp) being passed directly to the DOM element.

Reply:
https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes

`maxlength` shows the following warning in the dev console
```
Warning: Invalid DOM property `maxlength`. Did you mean `maxLength`?
```